### ISSUE TITLE
questions about training details

### ISSUE THREAD

**Issue by Pang-Yatian** [VAST-AI-Research/TriplaneGaussian#2]
Hi, Thanks for your great work!

May i ask how many GPU hours did you consume to train the model？

Could you please also reveal more training details such as batch size, learning rate, which cannot be found in the paper.

**Comment by zouzx** [comment]
We train the model on 8 GPUs for ~3 days for point cloud training and ~3 days for the triplane and 3D Gaussian training, with `bs=32` for each 80G GPU, and the learning rate is 4e-4 with warm-up cosine annealing.
